artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py

```diff
@@ -325,6 +325,14 @@
     "[no description] http/takeovers/netlify-takeover.yaml": "Domena jest skonfigurowana, aby serwować treści z narzędzia Netlify, ale domena docelowa jest wolna. Atakujący może potencjalnie zarejestrować taką domenę w serwisie Netlify aby umieścić tam swoje treści. Jeśli domena jest nieużywana, rekomendujemy jej usunięcie.",
     "WordPress Zoomsounds plugin 6.45 and earlier allows arbitrary files, including sensitive configuration files such as wp-config.php, to be downloaded via the `dzsap_download` action using directory traversal in the `link` parameter.": "Wtyczka WordPress o nazwie Zoomsounds w wersji 6.45 i wcześniejszych umożliwia atakującemu pobieranie dowolnych plików z serwera, w tym plików konfiguracyjnych."
     + UPDATE_HINT,
+    "TurboCRM contains a cross-site scripting vulnerability which allows a remote attacker to inject arbitrary JavaScript into the response returned by the application.": "Narzędzie TurboCRM zawiera podatność Cross-Site Scripting, umożliwiającą wstrzykiwanie dowolnego kodu JavaScript do odpowiedzi aplikacji."
+    + UPDATE_HINT,
+    "[no description] http/misconfiguration/elasticsearch.yaml": "Wykryto, że dane w systemie Elasticsearch są publicznie dostępne. Rekomendujemy zmianę tej konfiguracji.",
+    "The HTTP PUT method is normally used to upload data that is saved on the server at a user-supplied URL. If enabled, an attacker may be able to place arbitrary, and potentially malicious, content into the application. Depending on the server's configuration, this may lead to compromise of other users (by uploading client-executable scripts), compromise of the server (by uploading server-executable code), or other attacks.": "Wykryto, że metoda HTTP PUT jest włączona, co umożliwia atakującemu umieszczanie dowolnych plików na serwerze, co może doprowadzić m.in. do zdalnego wykonania kodu."
+    + RCE_EFFECT_DESCRIPTION,
+    "Microsoft Exchange Server is vulnerable to a remote code execution vulnerability. This CVE ID is unique from CVE-2021-31196, CVE-2021-31206.": "Wykryto, że Microsoft Exchange Server umożliwia zdalne wykonanie kodu." + RCE_EFFECT_DESCRIPTION + UPDATE_HINT,
```

Comment:
this translation is misleading as every Exchange Server is vulnerable, it should focus on version, eg `wersja obecnie używanego Microsoft Exchange Server jest podatna na CVE-2021-31196/CVE-2021-31206 i umożliwia zdalne wykonanie kodu`